components/mitsubishi_uart/mitsubishi_uart.cpp

```diff
@@ -133,6 +134,11 @@ void MitsubishiUART::update() {
   hp_bridge.sendPacket(GetRequestPacket::getStatusInstance());
   hp_bridge.sendPacket(GetRequestPacket::getCurrentTempInstance());
   )
+
+  // TODO: get this every 60 seconds instead of every n loops
+  if (this->_updateLoopCounter % 10 == 0 && !ts_bridge) {
```

Comment:
Don't check for `!ts_bridge` here, just send it anyway.  Most of the other packets we send are already redundant with an MHK2 attached.  If we want to try to cut down on bandwidth (is this a concern?), we should try to e.g. cache all received packets and only request new ones if we need them.  But I think that's a separate change.

Reply:
My primary concern here revolves around eventually overloading the heat pump with Too Many Thing At Once. 2400 baud is plenty for this use case, but it's still rather slow in general.

Assuming we keep requesting more and more information (especially so long as we transparently forward MHK packets), we might start making headaches for ourselves. c.f. specifically [akamali/mhk1_mqtt](https://github.com/akamali/mhk1_mqtt) noting the MHK1 has some rather strict timing.

See also #10 for a more broad scope, which might ultimately end with us maintaining our own state and not letting the thermostat talk to the heat pump directly (especially as Kumo does this).

I'm fine going either way here, but I also don't strictly see a need to have errors updated *as* frequently as active-state.

Reply:
EDIT: I'll just be lazy for now and send this every time. See inlined comments and above, but I also don't want to deal with thinking of a better implementation for this yet.

Reply:
I think the overloading concern is justified (I've already had some issues e.g. a20950dabd337557a59467d08f5783fd11bf3286 , and will leave open as requested), but I think the appropriate way to resolve is with a combination of minimum update periods, queue throttling, and caching.  Would rather resolve this all at once for all the packets instead of piecemeal.